### ISSUE TITLE
Intendation problem with Generic.WhiteSpace.ScopeIndent.IncorrectExact

### ISSUE THREAD

**Issue by roopemerikukka** [squizlabs/PHP_CodeSniffer#2043]
I'm getting this error:
```
Line indented incorrectly; expected 1 tabs, found 0
(Generic.WhiteSpace.ScopeIndent.IncorrectExact)
```

with these settings:

```xml
<arg name="tab-width" value="2"/>
<rule ref="Generic.WhiteSpace.ScopeIndent">
    <properties>
      <property name="indent" value="2"/>
    </properties>
  </rule>
```

So I'm trying to setup the configuration so that it uses spaces instead of tabs and the tab width equals to two spaces. I don't get why I'm getting error about missing tabs as I'm using spaces?

I'm using vscode with these settings if this is anyway helpful:
```json
{
  "editor.insertSpaces": true,
  "editor.detectIndentation": false,
  "editor.tabSize": 2
}
```

**Comment by gsherwood** [comment]
You must be including a standard in your ruleset that is enforcing tab indents on the ScopeIndent sniff. It would be doing this by including this rule:
```xml
<rule ref="Generic.WhiteSpace.ScopeIndent">
    <properties>
        <property name="tabIndent" value="true" />
    </properties>
</rule>
```
So you should try changing your existing ScopeIndent settings to this:
```xml
<rule ref="Generic.WhiteSpace.ScopeIndent">
    <properties>
        <property name="indent" value="2"/>
        <property name="tabIndent" value="false" />
    </properties>
</rule>
```

Also make sure you put this at the end of your ruleset.xml file so that it overrides any settings you are importing from other standards.

**Comment by roopemerikukka** [comment]
Okay the problem was with the configuration order. Thanks for help 👍
